quipucords/scanner/vcenter/inspect.py

```diff
@@ -339,15 +341,16 @@ def _property_set(self):
             all=False,
             type=vim.VirtualMachine,
             pathSet=[
-                'guest.net',
-                'name',
-                'runtime.host',
-                'summary.guest.hostName',
-                'summary.runtime.powerState',
-                'summary.config.guestFullName',
-                'summary.config.memorySizeMB',
-                'summary.config.numCpu',
-                'summary.config.uuid'
+                "guest.net",
+                "name",
+                "runtime.host",
+                "config.template",
```

Comment:
I can see you just added "config.template" here. Don't need to change this commit but consider in the future separating one commit reformatting old code and another one adding something new.